src/test/java/nitezh/ministock/dataaccess/RssFeedTests.java

```diff
@@ -0,0 +1,32 @@
+package nitezh.ministock.dataaccess;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.robolectric.RobolectricTestRunner;
+
+import java.net.HttpURLConnection;
+import java.net.URL;
+
+import static junit.framework.Assert.assertEquals;
+
+/**
+ * Created by nicholasfong on 2018-04-04.
+ */
+
+@RunWith(RobolectricTestRunner.class)
+public class RssFeedTests {
+    @Test
+    public void accessNasdaqRSSTest(){
+        URL url = null;
```

Comment:
redudant initializer. can remove `= null`

construct a test that should fail upon going to an invalid stock symbol
test is also too simple. construct a UI Automator test for RSS Feed click events